frontend/packages/volto-intranet/src/components/Footer/slots/FollowUsLogoAndLinks.tsx

```diff
@@ -0,0 +1,116 @@
+/**
+ * OVERRIDE: FollowUsLogoAndLinks
+ * REASON: Footer links should contains Feedback form link
+ * TICKET: https://gitlab.kitconcept.io/kitconcept/distribution-kitconcept-intranet/-/issues/124#note_21478
+ * ORIGINAL: @kitconcept/volto-light-theme
+ * FILE VERSION: 7.0.0a23
+ * DATE: 2025-08-25
+ * DEVELOPER: @iFlameing
+ */
+
+import { FormattedMessage, useIntl, defineMessages } from 'react-intl';
+import { useLocation } from 'react-router-dom';
+import cx from 'classnames';
+import { flattenToAppURL } from '@plone/volto/helpers/Url/Url';
+import ConditionalLink from '@plone/volto/components/manage/ConditionalLink/ConditionalLink';
+import { Container } from '@plone/components';
+import { useLiveData } from '@kitconcept/volto-light-theme/helpers/useLiveData';
+import LinkList from '@kitconcept/volto-light-theme/primitives/LinkList';
+import type {
+  PloneGobrSocialMediaSettings,
+  SiteFooterSettings,
+} from '@kitconcept/volto-light-theme/types';
+import type { Content } from '@plone/types';
+import SlotRenderer from '@plone/volto/components/theme/SlotRenderer/SlotRenderer';
+
+const messages = defineMessages({
+  feedback: {
+    id: 'Feedback about this page',
+    defaultMessage: 'Feedback about this page',
+  },
+});
+
+const FollowUsPostFooterLogoAndLinks = ({ content }: { content: Content }) => {
+  const social_links = useLiveData<
+    PloneGobrSocialMediaSettings['social_links']
+  >(content, 'plonegovbr.socialmedia.settings', 'social_links');
+
+  const intl = useIntl();
+  const location = useLocation();
+
+  let footer_links = useLiveData<SiteFooterSettings['footer_links']>(
+    content,
+    'voltolighttheme.footer',
+    'footer_links',
+  );
+  /* START CUSTOMIZATION */
+  footer_links = [
+    ...(footer_links || []),
+    {
+      '@id': '8b42f8f4-3d6e-4c97-9b8c-27c51e8b29a1',
+      href: [
+        {
+          '@id': `${location.pathname === '/' ? '' : location.pathname}/feedback-form`,
```

Comment:
@sneridagh Shouldn't this be added to the footer_links in the backend instead of customizing this component?

Reply:
@davisagli @sneridagh I tried to put inside the footerLinks in sidebar. But it doesn't allow me to put the relative link. That's why I have to customised this folder.

Reply:
Then we have to put afford into Universal link component and also widget because it doesn't allow to write a relative link. Maybe in another Pr.